src/store.js

```diff
@@ -72,6 +76,16 @@ const getProductsThunk = ()=>{
   }
 }
 
+const addToCart = (item) => {
+  return async (dispatch) => {
+    const respoanse = (await axios.post(`/api/products/${item.id}`, item)).data;
```

Comment:
spelling error 'response' instead of 'respoanse' on line 81 and 82

Reply:
I am sorry I will fix it later I am at duty now